modules/candidate_list/php/NDB_Menu_Filter_candidate_list.class.inc

```diff
@@ -127,7 +127,7 @@ class NDB_Menu_Filter_Candidate_List extends NDB_Menu_Filter
                 ON (ps.participant_status=pso.ID)
             LEFT JOIN Project p ON (c.ProjectID=p.ProjectID)
             LEFT JOIN subproject sp ON (s.SubprojectID=sp.SubprojectID)
-	    WHERE c.Active = 'Y'";
+            WHERE c.Entity_type = 'Human' and c.Active = 'Y'";
```

Comment:
The filter is still changed so that it always adds 'Human' when it shouldn't be unless that's selected in the filter.. this should just be c.Active='Y'